airflow/configuration.py

```diff
@@ -97,8 +98,6 @@ def default_config_yaml() -> dict:
 
     :return: Python dictionary containing configs & their info
     """
-    import airflow.utils.yaml as yaml
```

Comment:
This was intentional here and not at the module level -- importing cyaml is "relatively" slow, and I wanted to delay this unless its _needed_ as `airflow.configuration` is in the critical path right now when importing `airflow` (yes, it shouldn't be.), and this slows down all CLI commands.

Reply:
But `cyaml` is not imported in `airflow.utils.yaml` at the top level, only in functions. Importing `airflow.utils.yaml` only pulls in `sys` and `typing`, the yaml module is always only available when a function is called.

Reply:
Oh, look at that. I thought about that already :D

It probably didn't when I first wrote it but I changed it before PRing it. Or that is my excuse and I'm sticking with it.